ecommerce/templates/edx/error.html

```diff
@@ -13,7 +13,7 @@
         <div class="container">
             <div class="depth depth-2 message-error-content">
                 <h3>{{ error }}</h3>
-                {% trans "If you need assistance, contact onlinelearning@hms.harvard.edu." %}
+                {% trans "If you need assistance, contact <a href=\"mailto:onlinelearning@hms.harvard.edu\">onlinelearning@hms.harvard.edu</a>." %}
```

Comment:
I'm not sure about this change. It's affecting a translated block of text so will need updated translations. Perhaps you can keep the original text and pass it through [urlize](https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#urlize) instead?

Reply:
I did that because I didn't find any translations of this string anyway. `urlize` sounds like a good idea though, I'll do that. Thanks